src/coreclr/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt

```diff
@@ -41,6 +41,9 @@ instructionset     ,X86   ,Popcnt    ,        ,15 ,POPCNT   ,popcnt
 instructionset     ,X86   ,          ,        ,   ,Vector128,
 instructionset     ,X86   ,          ,        ,   ,Vector256,
 instructionset     ,X86   ,AvxVnni   ,        ,25 ,AVXVNNI  ,avxvnni
+instructionset     ,X86   ,          ,        ,   ,Sha1     ,
+instructionset     ,X86   ,          ,        ,   ,Sha256   ,
```

Comment:
I think the NativeAOT legs are failing because the managed name was not specified here (see the skipped column). Only Vector128/Vector256 can skip the column because they're weird.